pkg/destroy/bootstrap/bootstrap.go

```diff
@@ -41,7 +41,7 @@ func Destroy(dir string) (err error) {
 	// Set platform to azurestack after setting tfPlatformVarsFileName, because the
```

Comment:
Should we update this comment? I assume `platform = typesazure.StackCloud.Name()` returns the name of the specific azurestack in use.

Reply:
Fixed, but the new value is still a constant `AzureStackCloud` instead of `azurestack`